### DISCUSSION TITLE
Posibility of using Lighthouse Positioning system with ground targets

### DISCUSSION THREAD

**Discussion by peihan-li** [bitcraze/discussions#1525] [Q&A] [answered]
Hi, 

I have a question about extending the usage of the lighthouse positioning system with ground targets. Right now, we are doing research with aerial and ground vehicle coordination. Our lab doesn't have a fancy mocap system to provide the ground-truth location of the robots, so we proceeded with the "cheaper" version - the lighthouse system. It works well with the crazyflie drones, but can we adapt the ground vehicles to the same tracking system? Right now, our solution is to "glue" an inactive drone to the vehicle to get the position, but are there any fancy ways? Does the positioning deck work stand-alone?

Any discussion or suggestions will be appreciated!

Best,

Peihan

**Comment by peihan-li** [comment]
I just noticed I used "ground robots", "ground targets", and "ground vehicles" interchangeably. Sorry for any confusion, but you can consider it a robot car.

**Comment by knmcguire** [comment, marked as answer]
There are fancy ways for sure! but the question is if that is what you want.

Under 'development tab' of the lighthouse are instructions to directly interface the lighthouse deck with other hardware, however that means is that you'll need to retrieve the lighthouse timing signals on the other hardware along with the state estimate filtering that comes with it. We have a lot of c-files in the firmware that are made just for that purpose. So it will be quite some work on your end to get that to work (however there are some people here on discussions that are trying though). 

Easiest hands down is as you said, a barebone crazyflie with a lighthouse deck.
